src/components/Controls.vue

```diff
@@ -330,6 +337,22 @@ export default {
 		clickHideAddCardModel() {
 			this.showAddCardModal = false
 		},
+		setPageTitle(title) {
+			if (this.defaultPageTitle === false) {
+				this.defaultPageTitle = window.document.title
+				if (this.defaultPageTitle.indexOf(' - ' + t('spreed', 'Deck') + ' - ') !== -1) {
```

Comment:
```suggestion
				if (this.defaultPageTitle.indexOf(' - Deck - ') !== -1) {
```

The app name is currently not translated anywhere, so let's stick to that. Otherwise spreed would be the wrong app id here ;)